src/api/example/index.js

```diff
@@ -0,0 +1,54 @@
+import { okController } from '~/src/api/example/controllers/ok-controller'
+import {
+  getJsonController,
+  sendJsonController
+} from '~/src/api/example/controllers/json-controller'
+import { statusCodeController } from '~/src/api/example/controllers/status-code-controller'
+import { multipartController } from '~/src/api/example/controllers/multipart-controller'
+import { longPathController } from '~/src/api/example/controllers/long-path-controller'
+import { headerController } from '~/src/api/example/controllers/header-controller'
+
+export const exampleApi = {
+  plugin: {
+    name: 'exampleApi',
+    register: async (server) => {
+      server.route([
+        {
+          method: ['*'],
```

Comment:
nit: is it worth being explicit here rather than a wildcard? I always think of the wildcard o be used at the end as the catch all route.

https://hapi.dev/api/?v=21.3.12#server.route()

Maybe `['GET', 'POST', 'PUT', 'PATCH', 'DELETE', 'OPTIONS']`?